spec/components/candidate_interface/continuous_applications/course_review_component_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::ContinuousApplications::CourseReviewComponent do
+  describe 'course_fee' do
+    let(:application_choice) do
+      create(:application_choice,
+             current_course_option: create(:course_option, course: create(:course, fee_domestic:, fee_international:)))
+    end
+
```

Comment:
We may need some clarification on what should be shown if there are no fees to display. The Find service has a `#is_fee_based?` method that I don't think we have in Apply 🤦‍♂️ Do we need something similar or can we rely on presence only?

Reply:
we can just use `.fee?` as it's an enum.